pkg/kubectl/cmd/templates/command_groups.go

```diff
@@ -30,6 +34,9 @@ type CommandGroups []CommandGroup
 func (g CommandGroups) Add(c *cobra.Command) {
 	for _, group := range g {
 		for _, command := range group.Commands {
+			if command.PreRunE == nil || command.PersistentPreRunE == nil {
```

Comment:
No, commandgroup doesn't have anything to do with adding validation to a command… it shouldn't mutate the provided command